backend.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# main.py
+import sys
+import datetime
+
+from wsgiref.simple_server import make_server
+from pyramid.config import Configurator
+from pyramid.response import Response
+from pyramid.view import view_config
+from transport import http_transport
+
+@view_config(route_name='print_date')
+def print_date(request):
+    return Response(str(datetime.datetime.now()))
+
+def main(global_config, **settings):
```

Comment:
I believe the way you're calling this below (`main(sys.argv)`) means that `settings` is always going to be an empty dict, and you're not using `global_config` at all. It would probably be better to just make this function take 0 arguments. Otherwise you're going to have to include an argument parser of some sort to unpack the command line args (which always come in as a list, e.g. `[15, '--option=1', '-v']`) into a dict form that Pyramid's Configurator can accept.